firestore-counter/clients/ios/Sources/FirestoreCounter/FirestoreCounter.swift

```diff
@@ -0,0 +1,91 @@
+import FirebaseFirestore
+import Firebase
+
+public class FirestoreShardCounter {
```

Comment:
How about naming this `FirebaseDistributedCounter` - I think it's better to not expose the implementation strategy in the name. Plus, "Distributed Counter" is the extension's official name.